open-media-match/src/OpenMediaMatch/templates/index.html.j2

```diff
@@ -28,5 +28,25 @@
                 <td> <li> {{b['name']}} </li> </td>
             </tr>
         {% endfor %}
+        <h1>File Upload </h1>
```

Comment:
Suggest renaming this "Check Content For Matches" or something similar - we might add another file upload.